### PR TITLE
New analyzer: Backscatter.io

### PR DESCRIPTION
**Pull request by 9b**
Hi! I run Backscatter.io where we collect information about mass-scanning conducting on the Internet. This analyzer brings observations and enrichment data directly into TheHive and Cortex. I've tested various data-types in both systems with success. Apart from an API key, nothing else is required. Please let me know if there's anything needed from me to merge this!

screen shot 2019-02-08 at 6 58 11 pm
screen shot 2019-02-08 at 6 58 42 pm
screen shot 2019-02-08 at 6 58 51 pm

### CODE REVIEW

**Comment by saadkadhi** [comment]
Thanks @9b for your submission. We will review it as soon as possible and include it in the upcoming  Cortex-Analyzers 1.16.0 release (scheduled for early March).

**Comment by 9b** [comment]
Thank you!

File: analyzers/BackscatterIO/backscatter-io.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        """Use the Backscatter.io summary data to create a view."""
+        try:
+            taxonomies = list()
+            level = 'info'
+            namespace = 'Backscatter.io'
+
+            if self.service == 'observations':
+                summary = raw.get('results', dict()).get('summary', dict())
+                taxonomies = taxonomies + [
+                    self.build_taxonomy(level, namespace, 'Observations', str(summary.get('observations_count', 0))),
```
Comment (nadouani) [inline analyzers/BackscatterIO/backscatter-io.py]: Any reason these are made strings?

Comment (9b) [reply analyzers/BackscatterIO/backscatter-io.py]: No particular reason other than being safe.

**Pull request merged**